testsuites/CBLTester/CBL_Functional_tests/TestSetup_FunctionalTests/test_custom_conflicts.py

```diff
@@ -0,0 +1,1013 @@
+
+    if replicator_type == "pull":
+        for sg_doc in sg_docs_content:
+            doc_id = sg_doc["_id"]
+            cbl_doc = cbl_docs[doc_id]
+            assert sg_doc["sg_new_update1"] != cbl_doc["sg_new_update1"], "CCR failed to resolve conflict " \
+                                                                          "with delayed local win"
+            assert sg_doc["sg_new_update2"] != cbl_doc["sg_new_update2"], "CCR failed to resolve conflict " \
+                                                                          "with delayed local win"
+            assert sg_doc["sg_new_update3"] != cbl_doc["sg_new_update3"], "CCR failed to resolve conflict " \
+                                                                          "with delayed local win"
+            assert "random" not in cbl_doc, "CCR failed to resolve conflict with delayed local win"
+            assert "cbl_random" not in sg_doc, "CCR failed to resolve conflict with delayed local win"
+            assert new_docs_body[doc_id][1]["update_during_CCR"] == cbl_doc["update_during_CCR"], "CCR failed to " \
+                                                                                                  "resolve conflict " \
+                                                                                                  "with delayed " \
+                                                                                                  "local win"
+    elif replicator_type == "push_pull":
+        for sg_doc in sg_docs_content:
+            doc_id = sg_doc["_id"]
+            cbl_doc = cbl_docs[doc_id]
+            assert sg_doc["sg_new_update1"] == cbl_doc["sg_new_update1"], "CCR failed to resolve conflict " \
+                                                                          "with delayed local win"
+            assert sg_doc["sg_new_update2"] == cbl_doc["sg_new_update2"], "CCR failed to resolve conflict " \
+                                                                          "with delayed local win"
+            assert sg_doc["sg_new_update3"] == cbl_doc["sg_new_update3"], "CCR failed to resolve conflict " \
+                                                                          "with delayed local win"
+            assert "random" not in sg_doc, "CCR failed to resolve conflict with delayed local win"
+            assert "cbl_random" in sg_doc, "CCR failed to resolve conflict with delayed local win"
+            assert cbl_doc["update_during_CCR"] == sg_doc["update_during_CCR"], "CCR failed to resolve " \
+                                                                                "conflict with delayed" \
+                                                                                " local win"
+
+
+@pytest.mark.listener
+@pytest.mark.custom_conflict
+@pytest.mark.replication
+def test_stop_replicator_before_ccr_completes(params_from_base_test_setup):
+    """
+    @summary: resolve conflicts as per local doc
```

Comment:
Please add summary written in spreadhsheet

Reply:
see previous comments